stetho/src/main/java/com/facebook/stetho/inspector/network/AsyncPrettyPrinterRegistry.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.network;
+
+public interface AsyncPrettyPrinterRegistry {
+  public boolean register (String headerName, AsyncPrettyPrinterFactory factory);
+  public AsyncPrettyPrinterFactory lookup (String headerName);
```

Comment:
should probably add a `@Nullable` to this because it can return 'null' if something isn't registered for headerName. right?